test/tests/api/v2_0/nodes_tests.py

```diff
@@ -296,9 +296,11 @@ def test_node_workflows_get(self):
         for resp in resps:
             assert_not_equal(0, len(resp), message='No Workflows found for Node')
 
-    	Api().nodes_get_workflow_by_id('fooey')
-        resps_fooey = self.__get_data()
-    	assert_equal(len(resps_fooey), 0, message='Should be empty')
+        try:
+            Api().nodes_get_workflow_by_id('fooey')
+            fail(message='did not raise exception')
```

Comment:
Is it possible to indicate "what" did not raise the exception?   I'm just trying to imagine what information I will have if this fails to point me to what failed.